getOption.m

```diff
@@ -9,16 +9,16 @@
 % group of subjects to analyze
 opt.groups = {''}; % {'blnd', 'ctrl'};
 % suject to run in each group
-% opt.subjects = {[4:6]};  % {[1:2], [1:2]};
-opt.subjects = {[]};  % {[1:2], [1:2]};
+opt.subjects = {[4:6]};  % {[1:2], [1:2]};
+% opt.subjects = {[]};  % {[1:2], [1:2]};
 
 
 % task to analyze
-opt.taskName = 'MotionDecoding';
+% opt.taskName = 'MotionDecoding';
 
 % opt.taskName = 'visMotion';
 
-% opt.taskName = 'balloonanalogrisktask';
+opt.taskName = 'balloonanalogrisktask';
```

Comment:
Should I move all the information related to this dataset (ds001 on open neuro) out of getOption and use that to replace the current getOption template we have?

Reply:
it doesnt really matter. Any example that is avaialbe is good for the new user, and they will always have to change the taskName anyway.

Reply:
OK I will update the template then.